### PR TITLE
feat: 965 update calculate all endpoint

### LINKED ISSUE #965: Update calculate-all endpoint
Update the `calculate-all` endpoint so it calculate the mean geospatial data for parent trees _within_ selected orchards.

acceptance criteria:
- [ ] `mean_elevation` added
- [ ] `mean_long_degrees` added
- [ ] `mean_lat_degrees` added
- [ ] Refactor frontend to accommodate changes

### PR DESCRIPTION
**Pull request by craigyu**
# Description
Closes #965 #966

- [x] SMP mix geospatial calc verified
- [ ] Seedlot geospatial calc verified

Sorry for the huge PR, it's quite hard to write a good summary now that it's been a week since I started 💀
Feel free to ping me for a review session.
The calculation needs to verified still.
This PR also breaks the `seedlot_audit` table triggers.

#### New
- Mean geospatial calculation implementation
- Added related DTOs

#### Changed
- Combined the two endpoints `location-info` and `calculate-all` under `parent-trees/calculate`
- Updated frontend to accommodate the endpoint changes 

#### Removed
- `location-info` endpoint and its dependencies
- `calculate-all` endpoint and its dependencies

### How was this tested?
- [ ] 🧠 Not needed
- [x] 👀 Eyeball
- [x] 🤖 Added tests

###  What gif/image best describes this PR or how it makes you feel?

---
Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Frontend](https://nr-spar-46-frontend.apps.silver.devops.gov.bc.ca/)
[Backend](https://nr-spar-996-backend.apps.silver.devops.gov.bc.ca/actuator/health)
[Oracle-API](https://nr-spar-996-oracle-api.apps.silver.devops.gov.bc.ca/actuator/health)

Once merged, code will be promoted and handed off to following workflow run.
[Merge Workflow](https://github.com/bcgov/nr-spar/actions/workflows/merge.yml)

---

Thanks for the PR!

Deployments, as required, will be available below:
- [Frontend](https://nr-spar--frontend.apps.silver.devops.gov.bc.ca/)
- [Backend](https://nr-spar-996-backend.apps.silver.devops.gov.bc.ca/actuator/health)
- [Oracle-API](https://nr-spar-996-oracle-api.apps.silver.devops.gov.bc.ca/actuator/health)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/nr-spar/actions/workflows/analysis.yml)

After merge, new images are deployed in:
- [Merge Workflow](https://github.com/bcgov/nr-spar/actions/workflows/merge.yml)

---

Thanks for the PR!

Deployments, as required, will be available below:
- [Frontend](https://nr-spar-46-frontend.apps.silver.devops.gov.bc.ca/)
- [Backend](https://nr-spar-996-backend.apps.silver.devops.gov.bc.ca/actuator/health)
- [Oracle-API](https://nr-spar-996-oracle-api.apps.silver.devops.gov.bc.ca/actuator/health)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/nr-spar/actions/workflows/analysis.yml)

After merge, new images are deployed in:
- [Merge Workflow](https://github.com/bcgov/nr-spar/actions/workflows/merge.yml)

### CODE REVIEW

File: backend/src/main/resources/application.properties [hunk 1]
```diff
@@ -7,7 +7,7 @@ server.error.include-message=always
 
 # Actuator and ops
 management.endpoint.health.show-details = always
-spring.docker.compose.enabled = false
+spring.docker.compose.enabled = true
```
Comment (mgaseta) [inline backend/src/main/resources/application.properties:10]: Is this fixed now? I remember this was set to false due to some errors when running docker locally.

File: backend/src/test/java/ca/bc/gov/backendstartapi/endpoint/ParentTreeEndpointTest.java [hunk 2]
```diff
@@ -30,80 +31,234 @@ class ParentTreeEndpointTest {
...
+                .value(responseDto.smpMixMeanGeoData().getMeanLongitude().toString()))
+        .andExpect(
+            jsonPath("$.smpMixMeanGeoData.meanElevation")
+                .value(responseDto.smpMixMeanGeoData().getMeanElevation()))
         .andReturn();
   }
 
   @Test
-  @DisplayName("getLatLongElevationEmptyTest")
+  @DisplayName("Missing requst property error test")
```
Comment (mgaseta) [inline backend/src/test/java/ca/bc/gov/backendstartapi/endpoint/ParentTreeEndpointTest.java:217]: Typo: `request`

**Review by mgaseta — APPROVED** [review]
Nice work! Lots of good changes!
Added only one typo fix and a small question.

File: backend/src/main/resources/application.properties (hunk 1 above)
Comment (craigyu) [reply backend/src/main/resources/application.properties:10]: oops, changing it back to false

File: backend/src/test/java/ca/bc/gov/backendstartapi/endpoint/ParentTreeEndpointTest.java (hunk 2 above)
Comment (craigyu) [reply backend/src/test/java/ca/bc/gov/backendstartapi/endpoint/ParentTreeEndpointTest.java]: https://github.com/bcgov/nr-spar/pull/996/commits/c7570de126e97bc7804b30c7b3d5747730472920

File: backend/src/main/resources/application.properties (hunk 1 above)
Comment (craigyu) [reply backend/src/main/resources/application.properties]: https://github.com/bcgov/nr-spar/pull/996/commits/fd9c6cf506787f24da4bd30785b42baf7b8678ef

**Pull request merged**